terrawrap/utils/graph.py

```diff
@@ -0,0 +1,98 @@
+""" Holds graph utilities"""
+import os
+import networkx
+
+from typing import List, Tuple, Any
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "from typing import List, Tuple, Any" should be placed before "import networkx"](https://app.codacy.com/manual/amplify-education/terrawrap/pullRequest?prid=4739316)